src/viz/PointCloud.js

```diff
@@ -85,29 +136,31 @@ class PointCloud extends Core {
     });
   }
 
-  updatePointCloudGeometry(positions, colors) {
-    const { geometry } = this.object;
+  updatePointCloudGeometry(positions, colors, normals) {
+    const { geometry, material } = this.object;
+    if (
+      material.size !== this.options.size &&
+      !Number.isNaN(this.options.size)
+    ) {
+      material.size = this.options.size;
+      material.needsUpdate = true;
+    } else if (material.needsUpdate) {
```

Comment:
Why is this required? Won't this introduce unwanted bugs when something else other than change in size requires the material to update?

Reply:
Right. I kept it there to turn off re-renders on subsequent cycles, but I think this call can be safely moved to `this.updateOptions` instead to avoid the bugs.